Sources/Switchcraft/Switchcraft.swift

```diff
@@ -351,7 +351,13 @@ public class Switchcraft {
      * - returns: The new `UITapGestureRecognizer`.
      */
     private func makeDefaultGestureRecognizer(forVC viewController: UIViewController) -> UITapGestureRecognizer {
-        showSwitcher = { self.display(from: viewController) }
+        let weakVC = WeakViewController(weakReference: viewController)
```

Comment:
I don't think we need a wrapper in this case, since it's just being stored in a local variable in the closure, usually you only need wrapper types if you are storing in an array or other data structure, I think you can just use a weak capture in the capture list here:
```
showSwitcher = { [weak viewController] in
    if let viewController = viewController {
         self.display(from: viewController)
    }
}
```
and dispense with the wrapper class.

Reply:
Thanks @nbrooke I didn't know you can do that and it's great!